src/lib/components/icons/LabelInput.svelte

```diff
@@ -0,0 +1,23 @@
+<script>
```

Comment:
Because this component is reusable, it should be placed in `src/lib/components`. Also, please follow the lower caps file name and add this component to storybook!